### DISCUSSION TITLE
Moving to the new .NET Foundation CLA service

### DISCUSSION THREAD

**Discussion by dnfadmin** [reactiveui/ReactiveUI#3491] [General]
Hello everyone
The .NET Foundation is transitioning to a new CLA service and I'm posting here to let you know it's happening. Some background info:

The new CLA service is based off of https://github.com/microsoft/ContributorLicenseAgreement, except it uses the dotnet-policy-service app

- Current contributors do not need to sign the CLA again
- The CLA agreement itself is not changing
- As part of the install/configuration process, the following work items will be completed

1. Create a .github repo if it doesn't exist already
2. Install the dotnet-policy-service app
3. Create PRs for the platformcontext.xml and cla.yml files
4. Disable the existing CLA webhook
5. Push the PRs to enable the new CLA service

That's all there is to it. I'll update this issue once the work is complete

Thanks
Chris
.NET Foundation Operations team

**Comment by glennawatson** [comment]
Send us the install instructions in a email if possible. Given past issues we'd prefer to do the install ourselves.

**Reply by glennawatson** [reply]
Also not all projects in the reactiveui organisation are part of the .net foundation. We did it on a repository by repository basis in the original agreements so the .GitHub approach won't work here.

**Reply by rprouse** [reply]
You should be able to do it on a repo by repo basis by doing the updates in the `.github` directory of each repo, The org level repo just saves having to do it in each repo.

I'll reach out to Chris to see if there is better documentation or at least repos we can point to.
